[2024-12-25T06:27:27.605+0000] {processor.py:186} INFO - Started process (PID=13111) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:27:27.606+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:27:27.607+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:27:27.606+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:27:27.611+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:27:27.610+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:27:27.611+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:27:27.625+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.026 seconds
[2024-12-25T06:27:46.511+0000] {processor.py:186} INFO - Started process (PID=13156) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:27:46.512+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:27:46.513+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:27:46.513+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:27:46.518+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:27:46.517+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:27:46.519+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:27:46.533+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.027 seconds
[2024-12-25T06:28:13.946+0000] {processor.py:186} INFO - Started process (PID=13215) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:28:13.947+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:28:13.948+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:28:13.948+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:28:13.952+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:28:13.951+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:28:13.952+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:28:13.966+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.024 seconds
[2024-12-25T06:28:40.660+0000] {processor.py:186} INFO - Started process (PID=13259) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:28:40.664+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:28:40.666+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:28:40.665+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:28:40.671+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:28:40.670+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:28:40.672+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:28:40.694+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.042 seconds
[2024-12-25T06:29:38.460+0000] {processor.py:186} INFO - Started process (PID=13743) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:29:38.460+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:29:38.461+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:29:38.461+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:29:38.465+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:29:38.464+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:29:38.465+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:29:38.480+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.025 seconds
[2024-12-25T06:30:27.919+0000] {processor.py:186} INFO - Started process (PID=13937) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:30:27.920+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:30:27.921+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:30:27.921+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:30:27.924+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:30:27.923+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:30:27.925+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:30:27.939+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.024 seconds
[2024-12-25T06:30:51.783+0000] {processor.py:186} INFO - Started process (PID=14032) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:30:51.784+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:30:51.785+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:30:51.785+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:30:51.789+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:30:51.788+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:30:51.789+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:30:51.803+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.024 seconds
[2024-12-25T06:31:44.810+0000] {processor.py:186} INFO - Started process (PID=14181) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:31:44.811+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:31:44.812+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:31:44.812+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:31:44.815+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:31:44.814+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:31:44.816+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:31:44.830+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.025 seconds
[2024-12-25T06:32:39.545+0000] {processor.py:186} INFO - Started process (PID=14373) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:32:39.546+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:32:39.547+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:32:39.546+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:32:39.550+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:32:39.549+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:32:39.551+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:32:39.566+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.026 seconds
[2024-12-25T06:33:32.384+0000] {processor.py:186} INFO - Started process (PID=14611) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:33:32.384+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:33:32.386+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:33:32.385+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:33:32.390+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:33:32.389+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:33:32.390+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:33:32.404+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.025 seconds
[2024-12-25T06:34:25.213+0000] {processor.py:186} INFO - Started process (PID=14737) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:34:25.215+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:34:25.216+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:34:25.215+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:34:25.219+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:34:25.218+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:34:25.219+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:34:25.233+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.025 seconds
[2024-12-25T06:35:18.461+0000] {processor.py:186} INFO - Started process (PID=14825) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:35:18.462+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:35:18.464+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:35:18.463+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:35:18.467+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:35:18.466+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:35:18.468+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:35:18.483+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.027 seconds
[2024-12-25T06:36:06.805+0000] {processor.py:186} INFO - Started process (PID=15079) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:36:06.806+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:36:06.807+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:36:06.806+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:36:06.810+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:36:06.809+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:36:06.810+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:36:06.824+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.024 seconds
[2024-12-25T06:36:57.184+0000] {processor.py:186} INFO - Started process (PID=15474) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:36:57.185+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:36:57.186+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:36:57.186+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:36:57.190+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:36:57.189+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:36:57.190+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:36:57.203+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.024 seconds
[2024-12-25T06:37:47.868+0000] {processor.py:186} INFO - Started process (PID=15559) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:37:47.869+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:37:47.870+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:37:47.870+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:37:47.873+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:37:47.872+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:37:47.874+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:37:47.888+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.025 seconds
[2024-12-25T06:38:38.550+0000] {processor.py:186} INFO - Started process (PID=15641) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:38:38.551+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:38:38.552+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:38:38.552+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:38:38.556+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:38:38.555+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:38:38.556+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:38:38.569+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.024 seconds
[2024-12-25T06:39:31.309+0000] {processor.py:186} INFO - Started process (PID=15726) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:39:31.310+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:39:31.311+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:39:31.311+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:39:31.315+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:39:31.313+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
ModuleNotFoundError: No module named 'pipeline'
[2024-12-25T06:39:31.315+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:39:31.329+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.024 seconds
[2024-12-25T06:41:46.603+0000] {processor.py:186} INFO - Started process (PID=16006) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:41:46.604+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:41:46.606+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:41:46.606+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:41:46.612+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:41:46.610+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
  File "/workspace/openaq_de_project/pipeline/pipeline.py", line 2, in <module>
    from google.cloud import bigquery
ImportError: cannot import name 'bigquery' from 'google.cloud' (unknown location)
[2024-12-25T06:41:46.612+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:41:46.628+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.030 seconds
[2024-12-25T06:42:35.794+0000] {processor.py:186} INFO - Started process (PID=16285) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:42:35.795+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:42:35.797+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:42:35.796+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:42:35.801+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:42:35.799+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
  File "/workspace/openaq_de_project/pipeline/pipeline.py", line 2, in <module>
    from google.cloud import bigquery
ImportError: cannot import name 'bigquery' from 'google.cloud' (unknown location)
[2024-12-25T06:42:35.801+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:42:35.816+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.026 seconds
[2024-12-25T06:49:56.329+0000] {processor.py:186} INFO - Started process (PID=18910) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:49:56.330+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T06:49:56.332+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:49:56.332+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:49:56.337+0000] {logging_mixin.py:190} INFO - [2024-12-25T06:49:56.336+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 4, in <module>
    from pipeline.pipeline import run_pipeline
  File "/workspace/openaq_de_project/pipeline/pipeline.py", line 2, in <module>
    from google.cloud import bigquery
ImportError: cannot import name 'bigquery' from 'google.cloud' (unknown location)
[2024-12-25T06:49:56.338+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T06:49:56.353+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.028 seconds
[2024-12-25T07:15:20.422+0000] {processor.py:186} INFO - Started process (PID=23440) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:15:20.423+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T07:15:20.425+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:15:20.424+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:15:20.430+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:15:20.428+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 16, in <module>
    'start_date': date(2024, 12, 24),
                  ^^^^
NameError: name 'date' is not defined
[2024-12-25T07:15:20.431+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:15:20.450+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.034 seconds
[2024-12-25T07:16:16.389+0000] {processor.py:186} INFO - Started process (PID=23578) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:16:16.390+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T07:16:16.392+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:16:16.392+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:16:16.396+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:16:16.394+0000] {dagbag.py:387} ERROR - Failed to import: /home/gitpod/airflow/dags/pipeline_dag.py
Traceback (most recent call last):
  File "/workspace/openaq_de_project/airflow-env/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/gitpod/airflow/dags/pipeline_dag.py", line 16, in <module>
    'start_date': date(2024, 12, 24),
                  ^^^^
NameError: name 'date' is not defined
[2024-12-25T07:16:16.397+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:16:16.411+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.027 seconds
[2024-12-25T07:17:07.415+0000] {processor.py:186} INFO - Started process (PID=23763) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:17:07.416+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T07:17:07.418+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:17:07.418+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:17:07.427+0000] {processor.py:925} INFO - DAG(s) 'pipeline_dag' retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:17:07.517+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:17:07.517+0000] {override.py:1911} INFO - Created Permission View: can read on DAG:pipeline_dag
[2024-12-25T07:17:07.525+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:17:07.525+0000] {override.py:1911} INFO - Created Permission View: can edit on DAG:pipeline_dag
[2024-12-25T07:17:07.531+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:17:07.530+0000] {override.py:1911} INFO - Created Permission View: can delete on DAG:pipeline_dag
[2024-12-25T07:17:07.538+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:17:07.537+0000] {override.py:1911} INFO - Created Permission View: can read on DAG Run:pipeline_dag
[2024-12-25T07:17:07.543+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:17:07.542+0000] {override.py:1911} INFO - Created Permission View: can delete on DAG Run:pipeline_dag
[2024-12-25T07:17:07.548+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:17:07.547+0000] {override.py:1911} INFO - Created Permission View: can create on DAG Run:pipeline_dag
[2024-12-25T07:17:07.553+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:17:07.553+0000] {override.py:1911} INFO - Created Permission View: menu access on DAG Run:pipeline_dag
[2024-12-25T07:17:07.553+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:17:07.553+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-25T07:17:07.563+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:17:07.563+0000] {dag.py:3262} INFO - Creating ORM DAG for pipeline_dag
[2024-12-25T07:17:07.571+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:17:07.571+0000] {dag.py:4180} INFO - Setting next_dagrun for pipeline_dag to 2024-12-24 00:00:00+00:00, run_after=2024-12-25 00:00:00+00:00
[2024-12-25T07:17:07.585+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.174 seconds
[2024-12-25T07:18:00.868+0000] {processor.py:186} INFO - Started process (PID=23907) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:18:00.869+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T07:18:00.871+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:18:00.870+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:18:00.879+0000] {processor.py:925} INFO - DAG(s) 'pipeline_dag' retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:18:00.892+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:18:00.892+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-25T07:18:00.911+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:18:00.911+0000] {dag.py:4180} INFO - Setting next_dagrun for pipeline_dag to 2024-12-24 00:00:00+00:00, run_after=2024-12-25 00:00:00+00:00
[2024-12-25T07:18:00.928+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.066 seconds
[2024-12-25T07:18:53.156+0000] {processor.py:186} INFO - Started process (PID=23995) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:18:53.157+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T07:18:53.159+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:18:53.158+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:18:53.167+0000] {processor.py:925} INFO - DAG(s) 'pipeline_dag' retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:18:53.180+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:18:53.179+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-25T07:18:53.198+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:18:53.198+0000] {dag.py:4180} INFO - Setting next_dagrun for pipeline_dag to 2024-12-24 00:00:00+00:00, run_after=2024-12-25 00:00:00+00:00
[2024-12-25T07:18:53.214+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.063 seconds
[2024-12-25T07:19:48.400+0000] {processor.py:186} INFO - Started process (PID=24143) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:19:48.401+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T07:19:48.403+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:19:48.402+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:19:48.411+0000] {processor.py:925} INFO - DAG(s) 'pipeline_dag' retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:19:48.425+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:19:48.424+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-25T07:19:48.444+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:19:48.443+0000] {dag.py:4180} INFO - Setting next_dagrun for pipeline_dag to 2024-12-24 00:00:00+00:00, run_after=2024-12-25 00:00:00+00:00
[2024-12-25T07:19:48.459+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.064 seconds
[2024-12-25T07:20:41.558+0000] {processor.py:186} INFO - Started process (PID=24249) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:20:41.559+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T07:20:41.560+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:20:41.560+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:20:41.569+0000] {processor.py:925} INFO - DAG(s) 'pipeline_dag' retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:20:41.582+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:20:41.582+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-25T07:20:41.691+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:20:41.691+0000] {dag.py:4180} INFO - Setting next_dagrun for pipeline_dag to 2024-12-24 00:00:00+00:00, run_after=2024-12-25 00:00:00+00:00
[2024-12-25T07:20:41.705+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.153 seconds
[2024-12-25T07:21:38.684+0000] {processor.py:186} INFO - Started process (PID=24384) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:21:38.685+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T07:21:38.687+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:21:38.686+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:21:38.695+0000] {processor.py:925} INFO - DAG(s) 'pipeline_dag' retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:21:38.708+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:21:38.708+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-25T07:21:38.728+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:21:38.728+0000] {dag.py:4180} INFO - Setting next_dagrun for pipeline_dag to 2024-12-24 00:00:00+00:00, run_after=2024-12-25 00:00:00+00:00
[2024-12-25T07:21:38.743+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.065 seconds
[2024-12-25T07:22:33.988+0000] {processor.py:186} INFO - Started process (PID=24542) to work on /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:22:33.989+0000] {processor.py:914} INFO - Processing file /home/gitpod/airflow/dags/pipeline_dag.py for tasks to queue
[2024-12-25T07:22:33.991+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:22:33.990+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:22:33.999+0000] {processor.py:925} INFO - DAG(s) 'pipeline_dag' retrieved from /home/gitpod/airflow/dags/pipeline_dag.py
[2024-12-25T07:22:34.011+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:22:34.011+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-25T07:22:34.030+0000] {logging_mixin.py:190} INFO - [2024-12-25T07:22:34.030+0000] {dag.py:4180} INFO - Setting next_dagrun for pipeline_dag to 2024-12-24 00:00:00+00:00, run_after=2024-12-25 00:00:00+00:00
[2024-12-25T07:22:34.045+0000] {processor.py:208} INFO - Processing /home/gitpod/airflow/dags/pipeline_dag.py took 0.061 seconds
